### ISSUE TITLE
Configurable interval to reintroduce retired endpoints due to fail fast

### ISSUE THREAD

**Issue by owensk** [linkerd/linkerd#1213]
When setting a router to failFast, it would be nice to be able to also set an interval to attempt to reintroduce that end point back into rotation.

**Comment by adleong** [comment]
I agree that this would be good to have.  Finagle currently doesn't expose a way to override the default (https://github.com/twitter/finagle/blob/master/finagle-core/src/main/scala/com/twitter/finagle/service/FailFastFactory.scala#L33) so this would take some work to fix.
